package.json

```diff
@@ -136,6 +136,7 @@
   },
   "devDependencies": {
     "@babel/plugin-proposal-optional-chaining": "7.14.5",
+    "@babel/plugin-proposal-private-methods": "^7.18.6",
```

Comment:
Shall we remove the `^` to ensure that we match the version exactly, like we do with the rest of our dependencies?

Reply:
Absolutely, thanks for the catch!